Dockerfile

```diff
@@ -19,10 +19,11 @@ RUN apt-get update && apt-get install --no-install-recommends -y build-essential
     avrdude \
     && rm -rf /var/lib/apt/lists/*
 
-ENV keyboard=ergodox
-ENV subproject=ez
-ENV keymap=default
+ENV KEYBOARD=ergodox_ez
+ENV KEYMAP=default
 
-VOLUME /qmk
-WORKDIR /qmk
-CMD make clean ; make keyboard=${keyboard} subproject=${subproject} keymap=${keymap}
+VOLUME /qmk_firmware
+WORKDIR /qmk_firmware
+COPY . .
```

Comment:
The `COPY` is so that built images can be used in and of themselves, without needing to mount the host filesystem's `qmk_firmware` directory onto them.